src/components/shared/basic/Input/BasicInput.tsx

```diff
@@ -136,18 +136,21 @@ const BasicInput = ({
           onFocus={() => setFocus(true)}
         />
         <div className={'actions'}>
-          {(type === InputType.password || toggleHide) &&
-            (hidden ? (
-              <VisibilityIcon
-                style={{ marginLeft: '8px', color: Colors.secondary }}
-                onClick={toggleHidden}
-              />
-            ) : (
-              <VisibilityOffIcon
-                style={{ marginLeft: '8px', color: Colors.secondary }}
-                onClick={toggleHidden}
-              />
-            ))}
+          {
+            // eslint-disable-next-line
```

Comment:
Kindly provide an explanation for this disable linter

Reply:
Added disable linters here as we need OR condition here and by using ?? the logic was not working